File: Pensionsimulator.py
def pension_simulator():
    print("Välkommen till denna pensionssimulator")
    namn = input("Vad heter du i förnamn? ")
    alder = int(input("Hur gammal är du? "))
    ar_till_pension = 65 - alder
    print(f"Hej {namn}. Du går i pension om {ar_till_pension} år.")
    input("Tryck på en tangent för att avsluta...")

pension_simulator()

File: Pensionsimulator.cs
using System;

class PensionSimulator
{
    static void Main(string[] args)
    {
        Console.WriteLine("Välkommen till denna pensionssimulator");
        Console.Write("Vad heter du i förnamn? ");
        string namn = Console.ReadLine();
        Console.Write("Hur gammal är du? ");
        int alder = Convert.ToInt32(Console.ReadLine());
        int arTillPension = 65 - alder;
        Console.WriteLine($"Hej {namn}. Du går i pension om {arTillPension} år.");
        Console.Write("Tryck på en tangent för att avsluta...");
        Console.ReadKey();
    }
}